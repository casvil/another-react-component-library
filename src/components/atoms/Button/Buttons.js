import styled from "styled-components";

const Button = styled.button`
  padding: 12px 8px;
`;

export const PrimaryButton = styled(Button)`
  background-color: red;
`;
