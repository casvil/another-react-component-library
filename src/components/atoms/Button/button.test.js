import React from "react";
import { render } from "@testing-library/react";

import { PrimaryButton } from "./Buttons";

test("Renders a Primary Button", () => {
  const { container } = render(<PrimaryButton />);
  expect(container.firstChild).toMatchSnapshot();
});
