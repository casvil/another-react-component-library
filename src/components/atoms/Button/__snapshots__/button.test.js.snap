// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a Primary Button 1`] = `
<button
  class="sc-bdvvtL sc-gsDKAQ blxTGy dOiKxo"
/>
`;
